prow/config.yaml

```diff
@@ -10529,6 +10529,26 @@ periodics:
       image: gcr.io/k8s-testimages/kubekins-e2e:latest-master
       imagePullPolicy: Always
 
+- cron: '1 13 * * 2,4,6' # Run at 5:01PST (13:01 UTC) on even days
```

Comment:
Btw - you can run this more frequently than 'on even days' as they're small jobs. Sth like once every 6h or sth.